### PR TITLE
first 3 chapters updates, minor updates, mainly screenshots

### PR DESCRIPTION
**Pull request by ghtmtt**
### Description

Goal: First 3 chapter updates of Training Manual. Just minor updates, mainly screenshots and QGIS versions.

- [x] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [x] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

File: source/docs/training_manual/processing/intro.rst [hunk 1]
```diff
@@ -5,17 +5,34 @@
...
-
-For a more systematic description of all the framework components and their usage, it is recommended to check the corresponding chapter in the QGIS manual. Use it as a support text along with this guide.
-
-All the exercises in this guide use free data set that can be downloaded from the `QGIS website <https://qgis.org/downloads/data/>`_. The zip file to download contains several folders corresponding to each one of the lessons in this guide. In each of them you will find a QGIS project file. Just open it and you will be ready to start the lesson.
+This guide describes how to use QGIS processing framework. It assumes no
+previous knowledge of the processign framework or any of the applications that
```
Comment (DelazJ) [inline source/docs/training_manual/processing/intro.rst]: old typo
```suggestion
previous knowledge of the processing framework or any of the applications that
```

File: source/docs/training_manual/processing/intro.rst [hunk 2]
```diff
@@ -5,17 +5,34 @@
...
+it relies on. It assumes basic knowledge of QGIS. The chapters about scripting
+assume you have some basic knowledge of Python and maybe the QGIS Python API.
+
+This guide is designed for self-study or to be used for running a processing
+workshop.
+
+Examples in this guide use QGIS 3.4. They might not work or not be available in
+versions other than that ones.
```
Comment (DelazJ) [inline source/docs/training_manual/processing/intro.rst]:
```suggestion
versions other than that one.
```

File: source/docs/training_manual/processing/intro.rst [hunk 3]
```diff
@@ -5,17 +5,34 @@
...
+concept or some new element, which is indicated in the chapter title and the
+short introduction at the beginning of each chapter. That should make it easy to
+locate lessons dealing with a particular topic.
+
+For a more systematic description of all the framework components and their
+usage, it is recommended to check the corresponding chapter in the QGIS manual.
+Use it as a support text along with this guide.
+
+All the exercises in this guide use free data set that can be downloaded from
+the `QGIS website <https://qgis.org/downloads/data/>`_. The zip file to download
```
Comment (DelazJ) [inline source/docs/training_manual/processing/intro.rst]: Any idea the zip data concerned (there are two datasets in the link)? Are these data not moved to the qgis-training manual data repository?

Comment (ghtmtt) [reply source/docs/training_manual/processing/intro.rst]: you are right, link changed

File: source/docs/training_manual/processing/intro.rst [hunk 4]
```diff
@@ -5,17 +5,34 @@
...
+are more or less independent from each other, and each one introduces some new
+concept or some new element, which is indicated in the chapter title and the
+short introduction at the beginning of each chapter. That should make it easy to
+locate lessons dealing with a particular topic.
+
+For a more systematic description of all the framework components and their
+usage, it is recommended to check the corresponding chapter in the QGIS manual.
+Use it as a support text along with this guide.
+
+All the exercises in this guide use free data set that can be downloaded from
```
Comment (DelazJ) [inline source/docs/training_manual/processing/intro.rst]: May I suggest we avoid direct link here and refers to the [data section](https://docs.qgis.org/testing/en/docs/training_manual/foreword/foreword.html#data) at the beginning of the training manual? It's one more click for the reader but it's one less link to keep updated for writers, and translators if we change the branch or the repo in the future.
Something like (feel free to reword)
```suggestion
All the exercises in this guide use the same free dataset used throughout the
training manual and referenced at section :ref:`data_downloadlink`.
```

File: source/docs/training_manual/processing/intro.rst [hunk 5]
```diff
@@ -5,17 +5,34 @@
...
+the `QGIS website <https://github.com/qgis/QGIS-Training-Data/archive/v2.0.zip>`_.
```
Comment (DelazJ) [inline source/docs/training_manual/processing/intro.rst]:
```suggestion
```

File: source/docs/training_manual/processing/warning.rst [hunk 6]
```diff
@@ -22,7 +22,7 @@ The parameters dialog for that module is like the following one.
 
 .. image:: img/warning/kriging.png
 
-It look complex, right? 
+It look complex, right?
```
Comment (DelazJ) [inline source/docs/training_manual/processing/warning.rst]:
```suggestion
It looks complex, right?
```

File: source/docs/training_manual/processing/intro.rst [hunk 7]
```diff
@@ -5,17 +5,34 @@
...
-
-This guide is comprised of a set of small exercises of progressive complexity. If you have never used the processing framework, you should start from the very beginning. If you have some previous experience, feel free to skip lessons. They are more or less independent from each other, and each one introduces some new concept or some new element, which is indicated in the chapter title and the short introduction at the begining of each chapter. That should make it easy to locate lessons dealing with a particular topic.
-
-For a more systematic description of all the framework components and their usage, it is recommended to check the corresponding chapter in the QGIS manual. Use it as a support text along with this guide.
-
-All the exercises in this guide use free data set that can be downloaded from the `QGIS website <https://qgis.org/downloads/data/>`_. The zip file to download contains several folders corresponding to each one of the lessons in this guide. In each of them you will find a QGIS project file. Just open it and you will be ready to start the lesson.
+This guide describes how to use QGIS processing framework. It assumes no
```
Comment (havatv) [inline source/docs/training_manual/processing/intro.rst]:
```suggestion
This guide describes how to use the QGIS processing framework. It assumes no
```

File: source/docs/training_manual/processing/intro.rst [hunk 8]
```diff
@@ -5,17 +5,34 @@
...
+previous knowledge of the Processing framework or any of the applications that
+it relies on. It assumes basic knowledge of QGIS. The chapters about scripting
```
Comment (havatv) [inline source/docs/training_manual/processing/intro.rst]:
```suggestion
it rely on. It assumes basic knowledge of QGIS. The chapters about scripting
```

File: source/docs/training_manual/processing/intro.rst [hunk 9]
```diff
@@ -5,17 +5,34 @@
...
+assume you have some basic knowledge of Python and maybe the QGIS Python API.
+
+This guide is designed for self-study or to be used for running a processing
+workshop.
+
+Examples in this guide use QGIS 3.4. They might not work or not be available in
+versions other than that one.
+
+This guide is comprised of a set of small exercises of progressive complexity.
+If you have never used the processing framework, you should start from the very
```
Comment (havatv) [inline source/docs/training_manual/processing/intro.rst]:
```suggestion
If you have never used the processing framework, you should start from the
```

File: source/docs/training_manual/processing/intro.rst [hunk 10]
```diff
@@ -5,17 +5,34 @@
...
+beginning. If you have some previous experience, feel free to skip lessons. They
+are more or less independent from each other, and each one introduces some new
```
Comment (havatv) [inline source/docs/training_manual/processing/intro.rst]:
```suggestion
are more or less independent of each other, and each one introduces some new
```

File: source/docs/training_manual/processing/intro.rst [hunk 11]
```diff
@@ -5,17 +5,34 @@
...
+concept or some new element, which is indicated in the chapter title and the
```
Comment (havatv) [inline source/docs/training_manual/processing/intro.rst]:
```suggestion
concept or some new element, as indicated in the chapter title and the
```

File: source/docs/training_manual/processing/intro.rst [hunk 12]
```diff
@@ -5,17 +5,34 @@
...
+short introduction at the beginning of each chapter. That should make it easy to
+locate lessons dealing with a particular topic.
+
+For a more systematic description of all the framework components and their
+usage, it is recommended to check the corresponding chapter in the QGIS manual.
```
Comment (havatv) [inline source/docs/training_manual/processing/intro.rst]:
```suggestion
usage, it is recommended to check the corresponding chapter in the user manual.
```

File: source/docs/training_manual/processing/intro.rst (hunk 4 above)
Comment (havatv) [inline source/docs/training_manual/processing/intro.rst]:
```suggestion
All the exercises in use the free datasets that are used throughout the
training manual and referenced in section :ref:`data_downloadlink`.
```

File: source/docs/training_manual/processing/set_up.rst [hunk 13]
```diff
@@ -6,17 +6,16 @@ Setting-up the processing framework
...
 Later on we will show how to configure the external applications that are used
 for extending the list of available algorithms, but for now we are just going
 to work with the framework itself.
 
-The processing framework is a core QGIS plugin, which means that, if you are
-running QGIS 2.0 or later, it should already be installed in your system, since
-it is included with QGIS. In case it is active, you should see a menu called
-*Processing* in your menu bar. There you will find an access to all the
-framework components.
+The processing framework is a core QGIS plugin, which means that, it should
```
Comment (havatv) [inline source/docs/training_manual/processing/set_up.rst]:
```suggestion
The processing framework is a core QGIS plugin, which means that it should
```

File: source/docs/training_manual/processing/set_up.rst [hunk 14]
```diff
@@ -6,17 +6,16 @@ Setting-up the processing framework
...
+already be installed in your system, since it is included with QGIS. In case it
+is active, you should see a menu called *Processing* in your menu bar. There you
```
Comment (havatv) [inline source/docs/training_manual/processing/set_up.rst]:
```suggestion
is active, you should see a menu called :guilabel:`Processing` in your menu bar. There you
```

File: source/docs/training_manual/processing/set_up.rst [hunk 15]
```diff
@@ -6,17 +6,16 @@ Setting-up the processing framework
...
+will find an access to all the framework components.
```
Comment (havatv) [inline source/docs/training_manual/processing/set_up.rst]:
```suggestion
can access all the framework components.
```

File: source/docs/training_manual/processing/set_up.rst [hunk 16]
```diff
@@ -33,11 +32,8 @@ of the QGIS window.
 
 
 The toolbox contains a list of all the available algorithms, divided in groups
-so called *Providers*.
-Providers can be (de)activated in the settings dialog.
-A label in the bottom part of the toolbox will remind you of that whenever there
-are inactive providers. Use the link in the label to open the settings window
-and set up providers. We will discuss the settings dialog later in this manual.
+so called *Providers*. Providers can be (de)activated in the settings dialog. We
```
Comment (havatv) [inline source/docs/training_manual/processing/set_up.rst]:
```suggestion
called *Providers*. Providers can be (de)activated in the Processing part of the Options dialog
(:menuselection:`Settings --> Options...`). We
```

File: source/docs/training_manual/processing/set_up.rst [hunk 17]
```diff
@@ -33,11 +32,8 @@ of the QGIS window.
...
+will discuss the settings dialog later in this manual.
```
Comment (havatv) [inline source/docs/training_manual/processing/set_up.rst]:
```suggestion
will discuss that dialog later in this manual.
```

File: source/docs/training_manual/processing/warning.rst [hunk 18]
```diff
@@ -8,7 +8,7 @@ An important warning before starting
 Just like the manual of a word processor doesn't teach you how to write a novel
 or a poem, or a CAD tutorial doesn't show you how to calculate the size of a
 beam for a building, this guide will not teach you spatial analysis. Instead,
-it will show you how to use the QGIS processing framework, a powerful tool
+it will show you how to use the QGIS Processing framework, a powerful tool
 for performing spatial analysis, but it is up to you to learn the required
```
Comment (havatv) [inline source/docs/training_manual/processing/warning.rst]:
```suggestion
for performing spatial analysis. It is up to you to learn the required
```

File: source/docs/training_manual/processing/warning.rst [hunk 19]
```diff
@@ -8,7 +8,7 @@ An important warning before starting
...
 concepts that are needed to understand that type of analysis. Without them,
 there is no point on using the framework and its algorithms, although you
```
Comment (havatv) [inline source/docs/training_manual/processing/warning.rst]:
```suggestion
there is no point in using the framework and its algorithms, although you
```

**Comment by ghtmtt** [comment]
OK guys, I'm going to merge this if you don't mind. Thanks for the reviews!

**Pull request merged**